websites/V/VanMoof/presence.ts

```diff
@@ -0,0 +1,262 @@
+const presence = new Presence({
+		clientId: "941761536681201676"
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async function () {
+	const urlpath = window.location.pathname.split("/"),
+		presenceData: PresenceData = {
+			details: "Other",
+			largeImageKey: "logo",
+			startTimestamp: browsingTimestamp,
+			buttons: [
+				{
+					label: "View Page",
+					url: window.location.href.replace(`${urlpath[1]}/`, "")
+				}
+			]
+		};
+
+	switch (true) {
+		case /[a-z]{2}(-)[A-Z]{2}/g.test(urlpath[1]) && !urlpath[2]:
+			(presenceData.details = "Home"), delete presenceData.buttons;
```

Comment:
please dont do this, just add the {} and give each its own line

Reply:
Edited (https://github.com/PreMiD/Presences/pull/5925)